{{cookiecutter.project_slug}}/.envs/.production/.postgres

```diff
@@ -0,0 +1,2 @@
+POSTGRES_USER=
```

Comment:
I wonder if it would make sense to pre-populate these with random strings? Just like the secret key?

Reply:
This one's now addressed.